app/src/main/res/navigation/graph_main.xml

```diff
@@ -1,13 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Would prefer if the comment started after the xml decl.

Reply:
👍